static/src/javascripts/projects/common/modules/audio/utils.js

```diff
@@ -22,4 +24,28 @@ const range = (min: number, max: number) => {
     return ret;
 };
 
-export { format, formatTime, range };
+const sendToOphan = (id: string, eventName: string) => {
```

Comment:
The `audio:content:ready` event can also be sent using this function (index.js)

Reply:
Ah yes good point